src/export/bops/utils/index.ts

```diff
@@ -61,12 +62,6 @@ export const formatQuestion = ({
     ? fieldTitle
     : `[${schemaType} ${index + 1}] ${fieldTitle}`;
 
-export type SchemaResponses =
```

Comment:
Previously, this was repeated - we can actually infer this from the existing Zod schemas which should mean fewer changes in future.